### PR TITLE
Scripts/Northrend: improvements for quest Get Me Outa Here! (11673)

### PR DESCRIPTION
**Pull request by Wyreth**
**Changes proposed:**

Proper timers, creature texts, add missing gameobject and fix spawn position for questgiver.

**Target branch(es):**

- [x] 3.3.5
- [x] master

**Tests performed:** works fine.

### CODE REVIEW

File: sql/updates/world/3.3.5/9999_99_99_99_world_quest_get_me_outa_here.sql [hunk 1]
```diff
@@ -0,0 +1,32 @@
+--
+SET @GUID := xxx;
+DELETE FROM `gameobject` WHERE `guid`=@GUID;
+INSERT INTO `gameobject` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `position_x`, `position_y`, `position_z`, `orientation`, `rotation0`, `rotation1`, `rotation2`, `rotation3`, `spawntimesecs`, `animprogress`, `state`, `ScriptName`, `VerifiedBuild`) VALUES
+(@GUID, 182531, 571, 0, 0, 1, 1, 4413.98, 5368.06, -15.5602, 3.92534, -0, -0, -0.924195, 0.381922, 300, 100, 1, "", 0);
+UPDATE `creature` SET `position_x`=4414.223, `position_y`=5367.299, `position_z`=-15.49403, `orientation`=0.3665192, `spawntimesecs`=300 WHERE `id`=25589;
```
Comment (tkrokli) [inline sql/updates/world/3.3.5/9999_99_99_99_world_quest_get_me_outa_here.sql]: Recommendation: add a blank line for readability before
- `DELETE FROM`
- `UPDATE creature`

**Comment by Wyreth** [comment]
Added guid 1009 (currently unused, from TC-Unused-Guid-Search), feel free to change if not appropriate.

File: sql/updates/world/3.3.5/9999_99_99_99_world_quest_get_me_outa_here.sql (hunk 1 above)
Comment (Wyreth) [reply sql/updates/world/3.3.5/9999_99_99_99_world_quest_get_me_outa_here.sql]: Done!

File: src/server/scripts/Northrend/zone_borean_tundra.cpp [hunk 2]
```diff
@@ -1751,18 +1757,45 @@ class npc_bonker_togglevolt : public CreatureScript
...
-        void UpdateEscortAI(uint32 /*diff*/) override
+        void EnterCombat(Unit* who) override
         {
-            if (GetAttack() && UpdateVictim())
+            if (who->GetTypeId() != TYPEID_PLAYER)
+                if (roll_chance_i(20))
+                    Talk(SAY_BONKER_5);
+        }
+
+        void UpdateAI(uint32 diff) override
```
Comment (ccrs) [inline src/server/scripts/Northrend/zone_borean_tundra.cpp]: why the change from ```UpdateEscortAI``` to ```UpdateAI``` ?

Comment (Wyreth) [reply src/server/scripts/Northrend/zone_borean_tundra.cpp]: NPC must say an out of combat text every once in a while.

I'm not sure if UpdateEscortAI executes if the escort hasn't started yet. Is there any real difference between the two?

Comment (ccrs) [reply src/server/scripts/Northrend/zone_borean_tundra.cpp]: https://github.com/TrinityCore/TrinityCore/blob/3.3.5/src/server/game/AI/ScriptedAI/ScriptedEscortAI.cpp#L187

Comment (Wyreth) [reply src/server/scripts/Northrend/zone_borean_tundra.cpp]: Updated, thanks.

Comment (ccrs) [reply src/server/scripts/Northrend/zone_borean_tundra.cpp]: my pleasure, keep up the good work

**Comment by jackpoz** [comment]
@Wyreth could you please fix the merge conflicts and give it a try ingame too ?
Then I'd say it can be merged.

**Comment by jackpoz** [comment]
anyone would like to take over this ? otherwise I will close it as abandoned

**Comment by illfated** [comment]
All I can do is to maintain and update the content as recommended by others.
(Then again, I could open it in form of a new PR to do the same.)
I guess you prefer a real coder to deal with the script details.

**Comment by jackpoz** [comment]
Yeah, this PR needs someone willing to finish it, fix it, maintain it if an issue comes up after merging it. No big deal if we close it :)

**Comment by illfated** [comment]
Very well, then. I vote for closing it with the label **[Closed-Abandoned]** .

**Comment by illfated** [comment]
Oh very well then. I guess you also noticed the nopch error warning from the improved MapEntry assert commit (which is causing the nopch error here too).

**Comment by jackpoz** [comment]
Thanks for the PR :)
